app/controllers/api/v1/votes_controller.rb

```diff
@@ -22,6 +22,14 @@ def destroy
     end
   end
 
+  def index
```

Comment:
I added index, because rails was complaining that it didn'ts exist for get /votes?task_id=1290837

If there is a better way, show me the light :D